src/styles/_common/_typography.scss

```diff
@@ -1,181 +1,42 @@
-@mixin display-4() {
-  font-size: 112px;
-  margin-bottom: $gutter;
-}
+$base-font-size: 18; // Default font-size
```

Comment:
If you're starting a fresh baseline, why not going in REM's?
It's just my opinion :)